packages/core/src/html/NodeProcessor.ts

```diff
@@ -116,34 +133,41 @@ class NodeProcessor {
    * Layout element collection
    */
 
-  _collectLayoutEl(node, property) {
+  private static collectLayoutEl(node: DomElement): string | null {
     const $ = cheerio(node);
-    this[property].push($.html());
+    const html = $.html();
     $.remove();
+    return html;
```

Comment:
had to change this part functionally since `this[property]` is not going to pass tsc.